workloads/scale-perf/run_scale_fromgit.sh

```diff
@@ -4,10 +4,9 @@ set -x
 source ./common.sh
 source ../../utils/common.sh
 
-oc -n my-ripsaw delete benchmark/scale --ignore-not-found --wait
-
 # Scale up/down $_runs times
 for x in $(seq 1 $_runs); do
+  oc -n my-ripsaw delete benchmark/scale --ignore-not-found --wait
```

Comment:
Maybe delete all existing benchmarks in the ns instead of just scale to avoid grabbing wrong fields? Thoughts?

Reply:
my-ripsaw namespace is already deleted as a first step. Scale benchmark is the only one present.